agent/php_nrini.c

```diff
@@ -1730,6 +1752,55 @@ static PHP_INI_MH(nr_unsigned_int_mh) {
   return SUCCESS;
 }
 
+static PHP_INI_MH(nr_log_events_max_samples_stored_mh) {
+  nriniuint_t* p;
+  int val = NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED;
+  bool err = false;
+  nr_status_t parse_status = NR_SUCCESS;
+
+#ifndef ZTS
+  char* base = (char*)mh_arg2;
+#else
+  char* base = (char*)ts_resource(*((int*)mh_arg2));
+#endif
+
+  p = (nriniuint_t*)(base + (size_t)mh_arg1);
+
+  (void)entry;
+  (void)mh_arg3;
+  NR_UNUSED_TSRMLS;
+
+  /*
+   * -- An invalid value will result in the default value.
+   * -- A value < 0 will result in the default value
+   * -- A value > MAX will result in MAX value
+   */
+
+  p->where = 0;
+
+  if (0 != NEW_VALUE_LEN) {
+    parse_status = nr_strtol(&val, NEW_VALUE, 0);
+    if (0 > val || NR_FAILURE == parse_status) {
+      val = NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED;
+      err = true;
+    } else if (NR_MAX_LOG_EVENTS_MAX_SAMPLES_STORED < val) {
+      val = NR_MAX_LOG_EVENTS_MAX_SAMPLES_STORED;
+      err = true;
+    }
+    if (err) {
+      nrl_debug(NRL_INIT,
```

Comment:
I think it's better to issue `nrl_warning`. That's what's been done for `nr_loglevel_mh` when invalid value is detected.
```suggestion
      nrl_warning(NRL_INIT,
```

Reply:
resolved in 84867ece87b76e35e2b05d1458f8706cbcc3cc40